packages/stentor-models/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@xapp/config/tsconfig.json",
```

Comment:
Remove anything from "@xapp"?

Reply:
Some things will stay with @xapp prefix, like this shared config repository.